src/servicebus/azext_servicebus/_params.py

```diff
@@ -0,0 +1,408 @@
+                        ' deliveries. default value is 10.'),
+        c.argument('status', options_list=['--status'],
+                   arg_type=get_enum_type(['Active', 'Disabled', 'Restoring', 'SendDisabled', 'ReceiveDisabled',
+                                           'Creating', 'Deleting', 'Renaming','Unknown']),
+                   help='Enumerates the possible values for the status of a messaging entity.'),
+        c.argument('auto_delete_on_idle', options_list=['--auto-delete-on-idle'],
+                   validator=_validate_auto_delete_on_idle,
+                   help='ISO 8601 timeSpan idle interval after which the queue is automatically deleted. The minimum '
+                        'duration is 5 minutes.'),
+        c.argument('enable_partitioning', options_list=['--enable-partitioning'],
+                   validator=_validate_enable_partitioning,
+                   help='A boolean value that indicates whether the queue is to be partitioned across multiple message'
+                        ' brokers.'),
+        c.argument('enable_express', options_list=['--enable-express'],
+                   validator=_validate_enable_express,
+                   help='A boolean value that indicates whether Express Entities are enabled. An express queue holds a'
+                        ' message in memory temporarily before writing it to persistent storage.')
+        c.argument('forward_to', options_list=['--forward-to'],
+                   arg_type=name_type, help='Queue/Topic name to forward the messages'),
+        c.argument('forward_dead_lettered_messages_to', arg_type=name_type,
+                   options_list=['--forward-dead-lettered-messages-to'], help='Queue/Topic name to forward the'
+                                                                              ' Dead Letter message')
+
+
+    # region Queue Get
+    for scope in ['sb queue get', 'sb queue delete']:
+        with self.argument_context(scope) as c:
+            c.argument('resource_group_name', arg_type=resource_group_name_type, required=True)
+            c.argument('namespace_name', options_list=['--namespace-name'], required=True, help='name of the Namespace')
+            c.argument('queue_name', options_list=['--name', '-n'], required=True, help='Queue Name')
+
+
+    # region Queue Get
+    with self.argument_context('sb queue list') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, required=True)
+        c.argument('namespace_name', options_list=['--namespace-name'], required=True, help='name of the Namespace')
+
+
+    # region Queue Authorizationrule
+    for scope in ['sb queue authorizationrule create',
+                  'sb queue authorizationrule get',
+                  'sb queue authorizationrule delete',
+                  'sb queue authorizationrule listkeys',
+                  'sb queue authorizationrule regeneratekeys']:
+        with self.argument_context(scope) as c:
```

Comment:
You can simply set the scope to be `sb queue authorizationrule` then it will apply to everything underneath `sb queue authorizationrule`.

Reply:
Are you planning on addressing this or there's a reason to not change it?